general/documentation/contributing.md

```diff
@@ -100,9 +173,11 @@ even commit them and create a pull request.
 
 GitHub has also launched their own lightweight online editor which integrates tightly with GitHub. Take a look at [github.dev](https://github.dev/moodle/devdocs).
 
-### Installation
+### Local installation
```

Comment:
I think this section should appear before "Quick start with Gitpod" (because most of the developers would probably prefer to install it locally).

Apart from re-ordering these sections, I would also suggest adding a comment to the Gitpod section to clarify that's an alternative to local installations (otherwise users might think they also need it).